modules/tools/rode/main.tf

```diff
@@ -1,19 +1,71 @@
 resource "helm_release" "rode" {
-  repository = "https://harbor.toolchain.lead.prod.liatr.io/chartrepo/public"
-  timeout    = 120
+  repository = "https://rode.github.io/charts"
   name       = "rode"
   chart      = "rode"
   namespace  = var.namespace
-  version    = "0.4.0"
+  version    = "0.3.3"
+  wait       = true
+
+  set_sensitive {
+    name  = "grafeas-elasticsearch.grafeas.elasticsearch.username"
+    value = var.grafeas_elasticsearch_username
+  }
+
+  set_sensitive {
+    name  = "grafeas-elasticsearch.grafeas.elasticsearch.password"
+    value = var.grafeas_elasticsearch_password
+  }
 
   values = [
     templatefile("${path.module}/rode-values.tpl", {
-      iam_arn            = var.rode_service_account_arn
-      grafeas_cert       = "grafeas-cert"
-      rode_cert          = "rode-cert"
-      ingress_hostname   = "rode.${var.namespace}.${var.cluster}.${var.root_zone_name}"
-      localstack_enabled = var.localstack_enabled
+      ingress_enabled  = true
+      ingress_hostname = var.rode_ingress_hostname
+      ingress_annotations = {
+        "kubernetes.io/ingress.class" : var.ingress_class
+      }
+
+      oidc_config = {
+        enabled : var.oidc_issuer_url != "",
+        issuer : var.oidc_issuer_url,
+        requiredAudience : var.oidc_client_id,
+        roleClaimPath : "resource_access.${var.oidc_client_id}.roles",
+        tlsInsecureSkipVerify : false
+      }
+
+      grafeas_image_tag = var.grafeas_image_tag
     })
   ]
 }
 
+resource "helm_release" "rode_ui" {
+  count = var.ui_ingress_hostname == "" ? 0 : 1
```

Comment:
Is there a use case for installing the UI without an ingress? I'm thinking if I want to test this module locally I might install it without an ingress and port forward the service.

Reply:
Since the module is fairly purpose built I wouldn't think you'd want to use it outside of this architecture, and I thought we weren't really trying to make this config local-friendly, but I'm not opposed to switching back to an `enabled` variable for that flag. Thoughts?

Reply:
Yeah we probably will not use this module outside of the shared svc stage env but if we are going to have a feature flag here I think it should be purpose specific and enabling/disabled just the ui ingress on the hostname separately.